openqabot/types/incidents.py

```diff
@@ -109,8 +110,6 @@ def __call__(
                     full_post["openqa"]["VERSION"] = self.settings["VERSION"]
                     full_post["qem"]["version"] = self.settings["VERSION"]
                     full_post["openqa"]["DISTRI"] = self.settings["DISTRI"]
-                    full_post["openqa"]["_ONLY_OBSOLETE_SAME_BUILD"] = "1"
```

Comment:
We might still want to set this `if "_OBSOLETE" in settings`?

Reply:
oh geez, I removed too much. Thx. Reverted

Reply:
Also, the documentation is a little bit puzzling:
> _ONLY_OBSOLETE_SAME_BUILD | Only obsolete (or **deprioritize**) jobs for the same BUILD.

So it looks like it could be used with `_DEPRIORITIZEBUILD`, but on the other hand:

> Needs _OBSOLETE.